config/passport.js

```diff
@@ -1,26 +1,22 @@
-const passport = require("passport");
-const LocalStrategy = require("passport-local").Strategy;
-const mongoose = require("mongoose");
-const User = mongoose.model("User");
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
```

Comment:
Why are you using `require` instead of `import` which is the preferred ES6 syntax?